### ISSUE TITLE
can not run php artisan shaark:install on fresh git clone

### ISSUE THREAD

**Issue by ScrumpyJack** [MarceauKa/shaark#68]
hi there. Would like to try out Shaark but the installation fails 

Steps taken:

git clone https://github.com/MarceauKa/shaark
cd shaark/
export APP_ENV=local
composer install
cp .env.example .env
vim .env
php artisan key:generate
php artisan storage:link
php artisan shaark:install

error:

SQLSTATE[42000]: Syntax error or access violation: 1071 Specified key was too long; max key length is 767 bytes (SQL: alter table `users` add unique `users_email_unique`(`email`))

**Comment by ScrumpyJack** [comment]
Attempting any one of these fixes gets me further
https://stackoverflow.com/questions/42244541/laravel-migration-error-syntax-error-or-access-violation-1071-specified-key-wa

but i new get a sql statement error

```
   Illuminate\Database\QueryException  : SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'json not null, `custom_properties` json not null, `responsive_images` json not n' at line 1 (SQL: create table `media` (`id` bigint unsigned not null auto_increment primary key, `model_type` varchar(191) not null, `model_id` bigint unsigned not null, `collection_name` varchar(191) not null, `name` varchar(191) not null, `file_name` varchar(191) not null, `mime_type` varchar(191) null, `disk` varchar(191) not null, `size` bigint unsigned not null, `manipulations` json not null, `custom_properties` json not null, `responsive_images` json not null, `order_column` int unsigned null, `created_at` timestamp null, `updated_at` timestamp null) default character set utf8 collate 'utf8_unicode_ci' engine = InnoDB)
```

**Comment by MarceauKa** [comment]
Hi, thank you for your feedback. It's a well-know error. You should update to MySQL 5.7 or Maria 10.2. It seems you're using Maria 10.1.

**Comment by MarceauKa** [comment]
Fix for SQLite released in [1.2.40](https://github.com/MarceauKa/shaark/releases/tag/1.2.40) if you want to give a try.
